cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -460,17 +405,47 @@ func (oi *OVFImporter) CleanUp() {
 	}
 }
 
-func (oi *OVFImporter) importWithModule(osID string) error {
+func (oi *OVFImporter) importWithModule(osID string, diskInfos *[]ovfutils.DiskInfo) error {
```

Comment:
Since module import is the only option now, maybe rename this to just `import`?

Reply:
Changed to `importDisks` (The compiler complained that `import` is a reserved keyword)